### PR TITLE
Team user management for captains

### PR DESCRIPTION
**Pull request by halfa**

### CODE REVIEW

File: src/InsaLan/TournamentBundle/Controller/UserController.php [hunk 1]
```diff
@@ -416,6 +428,59 @@ public function leaveTeamAction($teamId) {
...
+
+        // captain cannot ban himself !
+        if($playerToBan !== $captain) {
+
+            $playerToBan->leaveTeam($team);
+            $team->removePlayer($playerToBan);
+
+            $em->persist($team);
+
+            $em->remove($playerToBan);
```
Comment (Lesterpig) [inline src/InsaLan/TournamentBundle/Controller/UserController.php]: Do you really want to remove the player? A persist shall be good

**Comment by Lesterpig** [comment]
Seems good, except for my comment. 

Why did you use the admin branch? It's not related to tournament administration?

Please be so kind to isolate trailing spaces fixes for next PRs ;)

File: src/InsaLan/TournamentBundle/Controller/UserController.php (hunk 1 above)
Comment (halfa) [inline src/InsaLan/TournamentBundle/Controller/UserController.php]: Can a player really be related to more than one team ? If not, the code above is correct as per Doctrine's doc (7.4). Else we need to check if `$playerToBan->getTeam()` is empty.

Comment (Lesterpig) [inline src/InsaLan/TournamentBundle/Controller/UserController.php]: Does it mean that the banned player will have to register from scratch to the tournament?
It shall be possible to only reset the team without removing the whole player.

Btw, it's just a tweak.

Comment (superboum) [inline src/InsaLan/TournamentBundle/Controller/UserController.php]: Need verification, but if you don't remove the player, he/shell will not have to reenter his username and other information. 
It was more critical when other information was League of Legend username verification, which was quite long (players have to create a rune page with a given name, wait 3 minutes, and validate it on insalan.fr to cheick they really own the summoner name they gave)

**Comment by halfa** [comment]
I used the `admin` branch because it's kind of related to tournament administration, even if it's user-side. The trailing spaces fix was merely a side-effect ;)

File: src/InsaLan/TournamentBundle/Controller/UserController.php (hunk 1 above)
Comment (halfa) [inline src/InsaLan/TournamentBundle/Controller/UserController.php]: Yes, but as the user interface doesn't allow you to create a player without joining a team, it's not handled by the template atm (if hang in step 1, the whole thing is canceled). So we will need to change this if we want to use the API again.

**Pull request merged**